source/_integrations/tedee.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: tedee
```

Comment:
```suggestion
title: Tedee
```

This needs to be adjusted / matched in Core too.

Reply:
The name should be upper case in core (like any other integration) but the manufacturer name on the device information is lower case (like in reality)

Reply:
matching this in core https://github.com/home-assistant/core/pull/106685